mm/2s2h/DeveloperTools/WarpPoint.cpp

```diff
@@ -107,6 +108,16 @@ void RegisterWarpPoint() {
 }
 
 void RenderWarpPointSection() {
+    bool skipToFileSelect = (bool)CVarGetInteger("gEnhancements.Cutscenes.SkipToFileSelect", 0);
+    UIWidgets::CVarCheckbox(
+        "Boot to Warp Point on Launch", WARP_POINT_CVAR "BootToWarpPoint",
+        UIWidgets::CheckboxOptions({ { .disabled = skipToFileSelect, .disabledTooltip = "Incompatible with Skip to File Select enhancement" } }),
+            .DefaultValue(true)
```

Comment:
clang format, I goofed again
```suggestion
        UIWidgets::CheckboxOptions({ { .disabled = skipToFileSelect,
                                       .disabledTooltip = "Incompatible with Skip to File Select enhancement" } })
        .DefaultValue(true)
```